drivers/storage/portworx/preflight.go

```diff
@@ -208,6 +155,14 @@ func (u *preFlightPortworx) RunPreFlight() error {
 		logrus.Infof("runPreflight: running pre-flight with existing PX-StoreV2 param, hard fail check enabled")
 	}
 
+	// Remove phone-home cm volume mount, does not exist yet
+	for i, volumeMount := range preflightDS.Spec.Template.Spec.Containers[0].VolumeMounts {
```

Comment:
Don't we need to remove the volume from the pod spec?

Reply:
Yes, we are running the CSI pod also and did plan on removing that later but don't want to that in this release.  So yeah when I look into that I can look into just using functions in the deployment code to just get what we need.   

The volume is there yeah but we don't have to remove the volume so long as its not mounted its ok.    But looking at the code now .. @harsh-px suggested trying "explore not adding the mounts in the first place".  So seeing if that can be done using the pre-flight annotation.

Reply:
@piyush-nimbalkar @harsh-px changed the code to use the pre-flight annotation to not add the phonehome configmap.     Let me know if you see any issues.  thx